azure_arc_servers_jumpstart/docs/aws_scale_ansible.md

```diff
@@ -0,0 +1,186 @@
+
+* [Create free AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+
+* [Install Terraform >=0.13](https://learn.hashicorp.com/terraform/getting-started/install.html)
+
+## Create Azure Service Principal (SP)   
+
+* To connect the AWS virtual machine to Azure Arc, an Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)). 
+
+    ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```az ad sp create-for-rbac -n "http://AzureArcAWS" --role contributor```
+
+    Output should look like this:
+
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcAWS",
+    "name": "http://AzureArcAWS",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    **Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)
+
+## Create an AWS identity
+
+In order for Ansible to dynamically generate a server inventory from AWS, we will need to create a new AWS IAM role with appropriate permissions.
+
+* Login to the [AWS management console](https://console.aws.amazon.com)
+
+* After logging in, click the "Services" dropdown in the top left. Under "Security, Identity, and Compliance" select "IAM" to access the [Identity and Access Management page](https://console.aws.amazon.com/iam/home)
+
+    ![](../img/aws_ubuntu/01.png) 
+
+    ![](../img/aws_ubuntu/02.png)
+
+* Click on "Users" from the left menu and then click on "Add user" to create a new IAM user.
+
+    ![](../img/aws_ubuntu/03.png)
+
+* On the "Add User" screen, give the user a name (e.g. "ansible") and select the "Programmatic Access" checkbox then click "Next"
+
+    ![](../img/aws_ubuntu/04.png)
+
+* On the next "Set Permissions" screen, select "Attach existing policies directly" and then check the box next to AmazonEC2FullAccess as seen in the screenshot then click "Next"
+
+    ![](../img/aws_ubuntu/05.png)
+
+* On the tags screen, assign a tag with a key of "azure-arc-demo" and click "Next" to proceed to the Review screen.
+
+    ![](../img/aws_ubuntu/06.png)
+
+* Double check that everything looks correct and click "Create user" when ready.
+
+    ![](../img/aws_ubuntu/07.png)
+
+* After the user is created, you will see the user's Access key ID and Secret access key. Copy these values down before clicking the Close button. In the screen below, you can see an example of what this should look like. Once you have these keys, you will be able to use them with Terraform to create AWS resources.
+
+    ![](../img/aws_ubuntu/08.png)
```

Comment:
You might want to re-create your terraform user or mask out these secrets

Reply:
yes those were already rotated out.